feconf.py

```diff
@@ -374,6 +374,14 @@ def get_empty_ratings():
 # Disables all the new structures' pages, till they are completed.
 ENABLE_NEW_STRUCTURES = False
 
+# Flag to disable sending emails related to reviews for suggestions. To be
```

Comment:
We really need fewer flags. If you can group these into a single flag, or introduce them only later, that would be helpful. At the moment all the flags for the feedback framework are interfering with each other and you need a matrix of test files.

Reply:
In fact since you don't use these anywhere I recommend just getting rid of them altogether.

Reply:
Ah, ignore please. This is already in develop. Not sure why it's showing up as an addition.